lib/plugins/aws/deploy/compile/events/apiGateway/lib/methods.js

```diff
@@ -5,13 +5,27 @@ const _ = require('lodash');
 
 module.exports = {
   compileMethods() {
+    const corsConfig = {};
     let endpointCounter = 1;
-
     _.forEach(this.serverless.service.functions, (functionObject, functionName) => {
       functionObject.events.forEach(event => {
         if (event.http) {
           let method;
           let path;
+          let corsEnabled = false;
+
+          const headers = [
+            'Content-Type',
+            'X-Amz-Date',
+            'Authorization',
+            'X-Api-Key',
+            'X-Amz-Security-Token'];
+
+          let cors = {
```

Comment:
Would be great if you can move this as well into the `if (!!event.http.cors)` section.